Mathlib/Data/Real/GoldenRatio.lean

```diff
@@ -117,6 +121,13 @@ theorem one_lt_gold : 1 < φ := by
   simp [← sq, gold_pos, zero_lt_one, - div_pow] -- Porting note: Added `- div_pow`
 #align one_lt_gold one_lt_gold
 
+theorem two_gt_gold : φ < 2 := by
+  rw [goldenRatio]
+  have h₀ : 3 ^ 2 = (9 : ℝ) := by norm_num
+  have h₁ : 3 = Real.sqrt 9 := by rw [← h₀, sqrt_sq]; linarith
+  have h₂ : Real.sqrt 5 < Real.sqrt 9 := by rw [sqrt_lt_sqrt_iff_of_pos]; repeat linarith
+  linarith
```

Comment:
1. we would call this `gold_lt_two` because of the direction of the inequality.
2. If you haven't learned about the `gcongr` tactic yet, let this be your introduction. It's very handy, and the `sqrt_lt'` allows us to get rid of the resulting `sqrt` lying around.
```suggestion
theorem gold_lt_two : φ < 2 := by calc
    (1 + sqrt 5) / 2 < (1 + 3) / 2 := by gcongr; rw [sqrt_lt'] <;> norm_num
    _ = 2 := by norm_num
```